keystore_test.go

```diff
@@ -182,60 +146,43 @@ func TestAliases(t *testing.T) {
 
 	sort.Strings(actualAliases)
 
-	if !reflect.DeepEqual(expectedAliases, actualAliases) {
-		t.Fatal("aliases must be equal")
-	}
+	assert.True(t, reflect.DeepEqual(expectedAliases, actualAliases), "aliases must be equal")
 }
 
 func TestLoad(t *testing.T) {
 	password := []byte{'p', 'a', 's', 's', 'w', 'o', 'r', 'd'}
 	defer zeroing(password)
 
 	f, err := os.Open("./testdata/keystore.jks")
-	if err != nil {
-		t.Fatalf("open test data keystore file: %s", err)
-	}
+	require.NoError(t, err)
 
 	defer func() {
-		if err := f.Close(); err != nil {
-			t.Fatalf("close test data keystore file: %s", err)
-		}
+		err := f.Close()
+		require.NoError(t, err)
 	}()
 
 	keyStore := New()
 
-	if err := keyStore.Load(f, password); err != nil {
-		t.Fatalf("decode test data keystore: %s", err)
-	}
+	err = keyStore.Load(f, password)
+	require.NoError(t, err)
 
 	actualPKE, err := keyStore.GetPrivateKeyEntry("alias", password)
-	if err != nil {
-		t.Fatalf("get private key entry: %s", err)
-	}
+	require.NoError(t, err)
 
 	expectedCT, err := time.Parse("2006-01-02 15:04:05.999999999 -0700 MST", "2017-09-19 17:41:00.016 +0300 EEST")
-	if err != nil {
-		t.Fatalf("parse creation time: %s", err)
-	}
+	require.NoError(t, err)
 
-	if !actualPKE.CreationTime.Equal(expectedCT) {
-		t.Errorf("unexpected private key entry creation time: '%v' '%v'", actualPKE.CreationTime, expectedCT)
-	}
+	assert.Truef(t, actualPKE.CreationTime.Equal(expectedCT),
+		"unexpected private key entry creation time: '%v' '%v'", actualPKE.CreationTime, expectedCT)
 
-	if len(actualPKE.CertificateChain) != 0 {
-		t.Errorf("unexpected private key entry certificate chain length: '%d' '%d'", len(actualPKE.CertificateChain), 0)
-	}
+	assert.Empty(t, actualPKE.CertificateChain, "unexpected private key entry certificate chain length")
 
 	pkPEM, err := os.ReadFile("./testdata/key.pem")
-	if err != nil {
-		t.Fatalf("read expected private key file: %s", err)
-	}
+	require.NoError(t, err)
 
 	decodedPK, _ := pem.Decode(pkPEM)
 
-	if !reflect.DeepEqual(actualPKE.PrivateKey, decodedPK.Bytes) {
-		t.Errorf("unexpected private key")
-	}
+	assert.True(t, reflect.DeepEqual(actualPKE.PrivateKey, decodedPK.Bytes), "unexpected private key")
```

Comment:
Please consider using assert.Equal and put expected and actual arguments in a correct order.